compiler-rt/lib/mpk_untrusted/alloc_site_handler.h

```diff
@@ -94,6 +95,14 @@ class StatsTracker {
   }
 };
 
+class PKeyInfo {
```

Comment:
how is this class to be used? If all fields are public, why not a struct?

Reply:
Changed to a struct.

Reply:
can we document what the struct is intended to track?

Reply:
Added documentation for struct's intended usage.